cmd/README.md

```diff
@@ -0,0 +1,521 @@
+# HarbourBridge: Spanner Evaluation and Migration
+
+[![cloudspannerecosystem](https://circleci.com/gh/cloudspannerecosystem/harbourbridge.svg?style=svg)](https://circleci.com/gh/cloudspannerecosystem/harbourbridge)
+
+HarbourBridge is a stand-alone open source tool for Cloud Spanner evaluation and
+migration, using data from an existing PostgreSQL, MySQL or DynamoDB database. 
+The tool ingests schema and data from either a pg_dump/mysqldump file or directly 
+from the source database, and supports both schema and data migration. For schema
+migration, HarbourBridge automatically builds a Spanner schema from the schema
+of the source database. This schema can be customized using the HarbourBridge
+schema assistant. For data migration, HarbourBridge creates a new Spanner
+database using the Spanner schema built during schema migration, and populates
+it with data from the source database.
+
+For more details on schema customization and use of the schema assistant, see
+[web/README](web/README.md). The rest of this README describes the command-line
+capabilities of HarbourBridge.
+
+HarbourBridge is designed to simplify Spanner evaluation and migration, and in
+particular for migrating moderate-size PostgreSQL/MySQL/DynamoDB datasets to Spanner 
+(up to about 100GB). Many features of PostgreSQL/MySQL, especially those that don't
+map directly to Spanner features, are ignored, e.g. stored functions and
+procedures, and sequences. Types such as integers, floats, char/text, bools,
+timestamps, and (some) array types, map fairly directly to Spanner, but many
+other types do not and instead are mapped to Spanner's `STRING(MAX)`.
+In the case of DynamoDB, the schema is inferred based on a certain 
+amount of sampled data.
+
+View HarbourBridge as a way to get up and running fast, so you can focus on
+critical things like tuning performance and getting the most out of
+Spanner. Expect that you'll need to tweak and enhance what HarbourBridge
+produces.
+
+Below are some quick starter examples on how to run HarbourBridge. You can look
+at specific sections for PostgreSQL, MySQL etc for detailed examples.
+
+For example, to generate Spanner schema from a PostgreSQL dump file, run
+
+```sh
+# Generate pg_dump file.
+pg_dump mydb > mydb.pg_dump
+
+harbourbridge schema -source=postgresql -source-profile="format=dump" < mydb.pg_dump
+# Or, (By default, format is "dump" when specifying input files)
+harbourbridge schema -source=postgresql < mydb.pg_dump
+# Or, you can specify full path to the dump file
+harbourbridge schema -source=postgresql -source-profile="format=dump,file=<full path to dump file>"
```

Comment:
nit: full path to dump file > absolute path to dump file?

Reply:
I didn't want to say absolute path since wasn't sure if that works for a file stored on GCS etc. Full path seemed more generic.

Reply:
Makes sense